### PR TITLE
1. Set gemm matrix size correctly.

### PR DESCRIPTION
**Pull request by jkottiku**
2. Remove sleep at intervals during test duration.

### CODE REVIEW

File: iet.so/src/iet_worker.cpp [hunk 1]
```diff
@@ -219,7 +212,7 @@ bool IETWorker::do_iet_power_stress(void) {
     rvs::action_result_t action_result;
     RSMI_POWER_TYPE type = RSMI_INVALID_POWER;
 
-    std::thread t(&IETWorker::blasThread, this, gpu_device_index, matrix_size_a, iet_ops_type, start, run_duration_ms,
+    std::thread t(&IETWorker::blasThread, this, gpu_device_index, matrix_size, iet_ops_type, start, run_duration_ms,
```
Comment (manoj-freyr) [inline iet.so/src/iet_worker.cpp:215]: good catch

File: iet.so/src/iet_worker.cpp [hunk 2]
```diff
@@ -176,26 +177,18 @@ void IETWorker::blasThread(int gpuIdx,  uint64_t matrix_size, std::string  iet_o
...
         //Print the gflops interval
         log_interval_gflops(gflops_interval);
+
         // check end test to avoid unnecessary sleep
         if (endtest)
             break;
-        //if gemm ops greater than 10000, lets yield
-        //if this is not happening we are ending up in
-        //out of memmory state
-        if(gem_ops > 10000) {
```
Comment (manoj-freyr) [inline iet.so/src/iet_worker.cpp:193]: why are we removing ?

File: iet.so/src/iet_worker.cpp [hunk 3]
```diff
@@ -176,26 +177,18 @@ void IETWorker::blasThread(int gpuIdx,  uint64_t matrix_size, std::string  iet_o
         iet_end_time = std::chrono::system_clock::now();
         //Duration in the call
         duration = time_diff(iet_end_time, iet_start_time);
-        gem_ops++;
```
Comment (manoj-freyr) [inline iet.so/src/iet_worker.cpp:179]: why are we removing?

File: iet.so/src/iet_worker.cpp (hunk 1 above)
Comment (manoj-freyr) [reply iet.so/src/iet_worker.cpp:215]: but what happens if we specify martix sizes a b and c, how can we ignore it ?

File: iet.so/src/iet_worker.cpp (hunk 2 above)
Comment (jkottiku) [reply iet.so/src/iet_worker.cpp:193]: This sleep results in unsustained GPU power utilization resulting in fluctuating power usage which is not expected why testing max. power cap testing.

File: iet.so/src/iet_worker.cpp (hunk 1 above)
Comment (manoj-freyr) [reply iet.so/src/iet_worker.cpp:215]: then this is a todo as a,b,and c can be used to override behavior and there is no need to have a b and c to be of same dimension

**Review by manoj-freyr — APPROVED** [review]

**Pull request merged**